microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -32,17 +32,25 @@ The various ways of loading local content into a WebView2 control support the fo
 | Additional web resources | ✔️ | ❌ | ✔️  | ✔️ |
 | Additional web resources resolved in WebView2 process | ✔️ | ❌ | ✔️ | ❌ |
 
+<!-- todo: add table row for custom scheme registration? -->
```

Comment:
add table row for custom scheme registration?
@aluhrs13 
rendered article section:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/working-with-local-content?branch=pr-en-us-3121&tabs=dotnetcsharp#selecting-an-approach

Reply:
no add'l col needed b/c 4th approach includes the new h4 section's approach.